### PR TITLE
Us321408 sessions pact

### PR DESCRIPTION
**Pull request by ppetekul**

### CODE REVIEW

File: access-checkout-ios-sdk-sessions.json [hunk 1]
```diff
@@ -0,0 +1 @@
...identity", "errorName": "fieldHasInvalidValue"}], "message": "The json body provided does not match the expected schema", "errorName": "bodyDoesNotMatchSchema"}, "matchingRules": {"$.body.validationErrors[0].message": {"match": "type"}, "$.body.message": {"match": "type"}}}}, {"request": {"body": {"cvc": "aaa", "identity": "identity"}, "path": "/sessions/payments/cvc", "method": "post", "headers": {"content-type": "application/vnd.worldpay.sessions-v1.hal+json", "Accept": "application/vnd.worldpay.sessions-v1.hal+json"}}, "description": " a request for a session reference with non-numerical CVV", "response": {"status": 400, "headers": {"Content-Type": "application/vnd.worldpay.sessions-v1.hal+json"}, "body": {"validationErrors": [{"message": "CVC must be numeric", "jsonPath": "$.cvc", "errorName": "fieldMustBeNumber"}], "message": "TThe json body provided does not match the expected schema", "errorName": "bodyDoesNotMatchSchema"}, "matchingRules": {"$.body.validationErrors[0].message": {"match": "type"}, "$.body.message": {"match": "type"}}}}], "consumer": {"name": "access-checkout-iOS-sdk"}, "metadata": {"pactSpecification": {"version": "2.0.0"}}, "provider": {"name": "sessions"}}
```
Comment (ochalet-wp) [inline access-checkout-ios-sdk-sessions.json:1]: I assume this is unused and can be removed?

File: access-checkout-ios-sdk-verified-tokens.json [hunk 2]
```diff
@@ -0,0 +1 @@
...arge"}], "message": "The json body provided does not match the expected schema", "errorName": "bodyDoesNotMatchSchema"}, "matchingRules": {"$.body.message": {"match": "type"}, "$.body.validationErrors[0].message": {"match": "type"}}}}, {"request": {"body": {"cardNumber": "notACardNumber", "cvc": "123", "identity": "identity", "cardExpiryDate": {"month": 1, "year": 2099}}, "path": "/verifiedTokens/sessions", "method": "post", "headers": {"content-type": "application/vnd.worldpay.verified-tokens-v1.hal+json"}}, "description": "a request with a PAN containing letters", "response": {"status": 400, "headers": {"Content-Type": "application/vnd.worldpay.verified-tokens-v1.hal+json"}, "body": {"validationErrors": [{"message": "Card number must be numeric", "jsonPath": "$.cardNumber", "errorName": "fieldHasInvalidValue"}], "message": "The json body provided does not match the expected schema", "errorName": "bodyDoesNotMatchSchema"}, "matchingRules": {"$.body.message": {"match": "type"}, "$.body.validationErrors[0].message": {"match": "type"}}}}], "consumer": {"name": "access-checkout-iOS-sdk"}, "metadata": {"pactSpecification": {"version": "2.0.0"}}, "provider": {"name": "verified-tokens"}}
```
Comment (ochalet-wp) [inline access-checkout-ios-sdk-verified-tokens.json:1]: I assume this is unused an can be removed?

File: scripts/upload_pact_VT.sh [hunk 3]
```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 set -e
 
+export PROJECT_VERSION="1.3.0"
```
Comment (ochalet-wp) [inline scripts/upload_pact_VT.sh:4]: - Do we need to increase the version?
- What about renaming this into `VERSION` or `CONTRACT_VERSION` ?

File: scripts/upload_pact_sessions.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+export PROJECT_VERSION="1.0.0"
```
Comment (ochalet-wp) [inline scripts/upload_pact_sessions.sh:4]: What about renaming this into `VERSION` or `CONTRACT_VERSION` ?

File: AccessCheckoutSDK/pacts/access-checkout-ios-sdk-sessions.json [hunk 5]
```diff
@@ -0,0 +1,119 @@
+{
+    "metadata": {
+        "pactSpecification": {
+            "version": "2.0.0"
```
Comment (ochalet-wp) [inline AccessCheckoutSDK/pacts/access-checkout-ios-sdk-sessions.json:4]: The pact specification version we use is 3.0.0, can you update it in the source code and regenerate the contarcts again? (and re-upload them on the broker then)

File: AccessCheckoutSDK/pacts/access-checkout-ios-sdk-verified-tokens.json [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+                }
+            }
+        }
+    ], 
+    "consumer": {
+        "name": "access-checkout-iOS-sdk"
+    }, 
+    "metadata": {
+        "pactSpecification": {
+            "version": "2.0.0"
```
Comment (ochalet-wp) [inline AccessCheckoutSDK/pacts/access-checkout-ios-sdk-verified-tokens.json:249]: Same thing for the version for this contract

File: scripts/upload_pact_VT.sh (hunk 3 above)
Comment (ppetekul) [reply scripts/upload_pact_VT.sh:4]: I have upgraded the version from 1.2.2 > 1.3.0 as we've added discovery test to the contract

File: AccessCheckoutSDK/pacts/access-checkout-ios-sdk-sessions.json (hunk 5 above)
Comment (ppetekul) [reply AccessCheckoutSDK/pacts/access-checkout-ios-sdk-sessions.json:4]: i found this line in the schemes:
scriptText = "${SRCROOT}/pact/bin/pact-mock-service start --pact-specification-version 2.0.0

it seems like we are using version 2.0.0? this is auto generated from what I can tell?

**Pull request merged**